test/orange/orangeinp/ObjectTestBase.hh

```diff
@@ -0,0 +1,111 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/orangeinp/ObjectTestBase.hh
```

Comment:
Why isn't this in test/orange?

Reply:
It is. The `file` directive is relative to the include base, which is `$SOURCE_DIR/test` 🙂 easy to miss, and it can make it confusing when we `#include "orange/orangeinp/ObjectTestBase.hh"`...

Reply:
Oh shoot